packages/fiber/tests/core/renderer.test.tsx

```diff
@@ -38,13 +43,21 @@ class HasObject3dMethods extends THREE.Object3D {
   }
 }
 
+class MyColor extends THREE.Color {
+  constructor(col: number) {
+    super(col)
+  }
+}
+
 extend({ HasObject3dMember, HasObject3dMethods })
 
 declare global {
   namespace JSX {
     interface IntrinsicElements {
       hasObject3dMember: ReactThreeFiber.Node<HasObject3dMember, typeof HasObject3dMember>
       hasObject3dMethods: ReactThreeFiber.Node<HasObject3dMethods, typeof HasObject3dMethods>
+      myColor: ReactThreeFiber.Node<MyColor, typeof MyColor>
+      group: Overwrite<ReactThreeFiber.Object3DNode<THREE.Group, typeof THREE.Group>, { args?: any }>
```

Comment:
Again not sure about this. I think it's actually confusing to add this?

Reply:
IMO `ts-ignore` that we had before may have caused other problems in typing, like:
```
//@ts-ignore
<group args={[n]} onPointerOver={1}>
  <group />
</group>
```
We might haven't even noticed that `onPointerOver` has a wrong value.
WDYT?

Reply:
Would you prefer a `myGroup` instead? We needed an `Object3D` that accepts `args` to test reconstruction behavior. It's indeed wrong to introduce wrong values to event props though, as that could silently pass/fail if the implementation changed (e.g. we make an assumption somewhere that it's a function if truthy).

Reply:
Yeah, great idea! Thanks!